daprdocs/content/en/operations/hosting/self-hosted/self-hosted-airgap.md

```diff
@@ -0,0 +1,74 @@
+---
+type: docs
+title: "How-To: Run Dapr in an offline or airgap environment"
+linkTitle: "Run in offline or airgap"
+weight: 30000
+description: "How to deploy and run Dapr in self-hosted mode in an airgap environment"
+---
+
+## Overview
+
+By default, Dapr initialization downloads binaries and pulls images from the network to setup the development environment. However, Dapr also supports offline or airgap installation using pre-downloaded artifacts, either with a Docker or slim environment. The artifacts for each Dapr release are built into a Dapr Installer Bundle which can be downloaded from the release. By using this installer bundle with the Dapr CLI `init` command, you can install Dapr into environments that do not have any network access.
+##  Setup
+
+Before airgap initialization, it is required to download a Dapr Installer Bundle beforehand, containing the CLI, runtime and dashboard packaged together. This eliminates the need to download binaries as well as Docker images when initializing Dapr locally.
+
+1.  Download the [Dapr Installer Bundle](https://github.com/dapr/installer-bundle/releases) for the specific release version. For example, daprbundle_linux_amd64.tar.gz, daprbundle_windows_amd64.zip.
+2. Unpack it.
+3. To install Dapr CLI copy the `daprbundle/dapr (dapr.exe for Windows)` binary to the desired location:
+   * For Linux/MacOS - `/usr/local/bin`
+   * For Windows, create a directory and add this to your System PATH. For example create a directory called `c:\dapr` and add this directory to your path, by editing your system environment variable.
+
+ ## Initialize Dapr environment
+
+Dapr can be initialized in an airgap environment with or without Docker containers.
+ ### Initialize Dapr with Docker
+
+([Prerequisite](#Prerequisites): Docker is available in the environment)
+
+Move to the bundle directory and run the following command:
+``` bash
+dapr init --from-dir .
```

Comment:
Add a note that if the cli is not moved to the desired location and not added in path, they can use the local dapr binary in the bundle. The steps above is to move it to the usual location and add it to path .This is just if they do not do it ....

Reply:
Done